### ISSUE TITLE
i can't see Mov2mov plugin on stable-diffusion-webui, how i fix it

### ISSUE THREAD

**Issue by roborobo3333** [Scholar01/sd-webui-mov2mov#154]
![Screenshot 2024-06-10 160123](https://github.com/Scholar01/sd-webui-mov2mov/assets/172255289/ec1b7cb3-a3f4-4260-af3e-73ffd33a0b87)

**Comment by devIndustrial** [comment]
Exactly the same problem, as I understand it, the author gave up on this plugin and no longer supports it.

**Comment by Webster-Wu** [comment]
same problem

**Comment by gizemgizg** [comment]
I solved the not showing problem. You can use this ✨ @Webster-Wu  @devIndustrial @roborobo3333 

https://github.com/gizemgizg/sd-webui-mov2mov

**Comment by titlestad** [comment]
> I solved the not showing problem. You can use this ✨ @Webster-Wu @devIndustrial @roborobo3333
> 
> https://github.com/gizemgizg/sd-webui-mov2mov

No, your version isn't showing either.

**Comment by gizemgizg** [comment]
[SD version ](version: v1.9.4)version: [v1.9.4](https://github.com/AUTOMATIC1111/stable-diffusion-webui/commit/feee37d75f1b168768014e4634dcb156ee649c05)

<img width="1436" alt="Ekran Resmi 2024-07-29 14 34 29" src="https://github.com/user-attachments/assets/c94455c5-03ec-480f-92e4-e41fa4695785">

**Comment by eschool95** [comment]
> [SD version ](version: v1.9.4)version: [v1.9.4](https://github.com/AUTOMATIC1111/stable-diffusion-webui/commit/feee37d75f1b168768014e4634dcb156ee649c05)
> 
> <img alt="Ekran Resmi 2024-07-29 14 34 29" width="1436" src="https://private-user-images.githubusercontent.com/64318805/353028506-c94455c5-03ec-480f-92e4-e41fa4695785.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjI1MzI2NzMsIm5iZiI6MTcyMjUzMjM3MywicGF0aCI6Ii82NDMxODgwNS8zNTMwMjg1MDYtYzk0NDU1YzUtMDNlYy00ODBmLTkyZTQtZTQxZmE0Njk1Nzg1LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA4MDElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwODAxVDE3MTI1M1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWViMGFiYmQyNmQwOWEwMzM5MTllMjhlZmU0ZTY0NjlmN2EyZjRlNGQxYjE1MDAwM2I0OGUxNTg3NDBlNTEzYWMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.wX-gTK-Sv-CpXSAjtu_BQ_ak8J5Tn1XBGcuoovgx_0Q">

tried your fork as well and still wont pop up. unsure how to get it back after update.

**Comment by gizemgizg** [comment]
Have you tried deleting the old version from the sd extension directory and installing the mov2mov version I shared? works fine for me

**Comment by viatonx** [comment]
> I solved the not showing problem. You can use this ✨ @Webster-Wu @devIndustrial @roborobo3333
> 
> https://github.com/gizemgizg/sd-webui-mov2mov

Thank you gizem. It worked this way

**Comment by tglunzer01** [comment]
Just want to confirm. My Stable Diffusion said mov2mov was allread_y installed. But there was no tab. I took above advice and found the extensions folder with the mov2mov extension folder,
automatic1111.git\app\extensions
found the move2mov folder
deleted that folder. Then followed normal Install from URL instructions and pasted the github code link.
https://github.com/Scholar01/sd-webui-mov2mov.git
Install
Then Apply and Restart UI
Quit Stable Diffusion, then restarted
After re-start the tab appeared.
![movemove tab](https://github.com/user-attachments/assets/70b2ad66-eb0d-4ff0-bc59-cc7d213417a1)

**Comment by emotionaldreams** [comment]
for me it corrupts the extra networks and extensions areas. neither show anything. no cards show in extra networks. and extension tab is tuck on "loading". so extentions installed don't show. soon as i move the mov2mov folder out, everything goes back to normal

**extra networks**
![Screenshot (4)](https://github.com/user-attachments/assets/f8e1716d-825b-414c-bdcf-da67376f8ca2)


**extentions**
![Screenshot (5)](https://github.com/user-attachments/assets/697f93d2-2c66-4a66-9794-4dca511938f2)

**Comment by gizemgizg** [comment]
https://github.com/Scholar01/sd-webui-mov2mov/issues/154#issuecomment-2255690686

@emotionaldreams

**Comment by emotionaldreams** [comment]
> [#154 (comment)](https://github.com/Scholar01/sd-webui-mov2mov/issues/154#issuecomment-2255690686)
> 
>

yes urs worked :) thanks.  but the video codec selection  area is missing

**Comment by tedtninety** [comment]
@gizemgizg 
tab shows . Added video, and face and get this error;     inputs_cached = block.data_model(**inputs_cached)  # type: ignore
TypeError: gradio.components.label.LabelData() argument after ** must be a mapping, not str

**Comment by Pralay9** [comment]
@tedtninety same problem. is it still not solved?

**Comment by Scholar01** [comment]
It has been fixed. If you have other problems, please submit a new issue.
